Docs/CommonErrors.md

```diff
@@ -0,0 +1,70 @@
+# Common Wallet Library Errors
+
+We return a Result object for every external interface that could have the potential to throw. Each public interface and what error could occur is outlined here.
+
+## Outer Errors
+### `VerifiedIdClient`
+
+`createVerifiedIdRequest`
```

Comment:
There may also be `MalformedRequestError` as a matter of "We understood but could not create a request from it, please contact whoever you got this request from" rather than "You need to add support if you want this to work."